src/lxc/tools/arguments.c

```diff
@@ -215,9 +216,14 @@ extern int lxc_arguments_parse(struct lxc_arguments *args, int argc,
 			break;
 		case 'o':
 			args->log_file = optarg;
+			logfile = true;
 			break;
 		case 'l':
 			args->log_priority = optarg;
+			if (!logfile &&
+			    args->log_file &&
+			    strncmp(args->log_file, "none", strlen(args->log_file)) == 0)
+				args->log_file = NULL;
```

Comment:
Why is all of that necessary? Shouldn't we just pass through `none` to `lxc_log_enable_logfile()` in `log.c`. It handles the `none` case, right?

Reply:
@brauner "lxc-unshare -o noneasdfasdfasdf" is also working. Is it allowed?